### DISCUSSION TITLE
Add The "Viewed" Button for a file to the bottom as well

### DISCUSSION THREAD

**Discussion by deve-sh** [community/community#27465] [Pull Requests] [answered]
Many times when reviewing Pull Requests on both Web, as well as the App, the file being reviewed, might be lengthy, a "[Viewed](https://github.blog/2019-07-01-mark-files-as-viewed/)" button at the bottom of the file would make it easier for reviewers, by not having to scroll all the way up to the beginning of the file in the file list in order to mark a file as viewed.

**Comment by jfullstackdev** [comment, marked as answer]
this is from Chrome, even if it is too large, it's always there 

![image](https://user-images.githubusercontent.com/47092464/181573464-4c30faf3-c7a2-42c7-a213-e66ce3cca4e3.png)

you don't need to scroll up just to see that

**Comment by deve-sh** [comment]
Really sorry for the confusion, I guess I meant the mobile app could use the same functionality.

**Reply by jfullstackdev** [reply]
not very sure of the mobile app, but I'm seeing posts that some features are actually not present, but to solve the issue, say, you only have access to a smartphone, log in too in any browser installed on that phone (but hopefully it's Chrome because others are having problem too) and hopefully, it's the same thing that we see on the standard browser

there was one post also about one feature is not available and it is the solution, to log in in a standard browser installed on that phone

but yes this is always a valid request and hopefully will implement this in the future
